billing-service/src/main/java/com/petclinic/billing/datalayer/DataSetupService.java

```diff
@@ -22,11 +22,11 @@ public DataSetupService(BillService billService) {
     public void run(String... args) throws Exception {
 
 
-        BillRequestDTO b1 = new BillRequestDTO( "1", "general", "1", LocalDate.of(2021,9,19),59.99,BillStatus.PAID);
-        BillRequestDTO b3 = new BillRequestDTO( "3", "operation", "1", LocalDate.of(2021,9,21), 199.99,BillStatus.PAID);
-        BillRequestDTO b4 = new BillRequestDTO( "4", "injury", "1",  LocalDate.of(2021,9,22), 199.99,BillStatus.UNPAID);
-        BillRequestDTO b2 = new BillRequestDTO( "2", "operation", "2", LocalDate.of(2021,9,20), 199.99,BillStatus.OVERDUE);
-        BillRequestDTO b5 = new BillRequestDTO( "5", "chronic", "3",  LocalDate.of(2021,9,23), 199.99,BillStatus.UNPAID);
+        BillRequestDTO b1 = new BillRequestDTO( "1", "general", "1", LocalDate.of(2023,9,19),59.99,BillStatus.PAID, LocalDate.of(2023, 10,3));
```

Comment:
Good idea of updating the dates of the bills to more recently so that the due dates make more sense!